addons/website/controllers/main.py

```diff
@@ -799,6 +800,14 @@ def actions_server(self, path_or_xml_id_or_id, **post):
 
         return request.redirect('/')
 
+    # ------------------------------------------------------
+    # Video information
+    # ------------------------------------------------------
+
+    @http.route(['/website/get_video_thumbnail_url'], type='json', auth='user', website=True)
+    def get_video_thumb_url(self, video_url):
+        return get_video_thumbnail_url(video_url)
```

Comment:
The `auth="user"` protection is probably good enough but maybe we can go further and check if the user has the restricted editor rights? :pray: Otherwise it means that portal users can use any Odoo instance to fetch third party video thumbnails... probably not that big of a deal but :shrug: